config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  get "/prizes", to: "prizes#index"
-  get "/configurations", to: "wheel_configurations#index"
+  get '/wheel', to: 'wheel#show'
```

Comment:
This won't work until you actually change the `show` method to always return the same thing. The show expects an `id` to be passed, and this case you are not passing anything.